### ISSUE TITLE
Fix pyflakes for manhole and tap

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7001]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#7001|
|Type|enhancement|
|Created|2014-02-27 13:51:12Z|
|Branch|https://github.com/twisted/twisted/tree/pyflakes-tap-manhole-7001|

Attachments:

* [7001-1.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/861/8611389f952d506d221dc8385f476c3f28903cdd/ac24d64fed39ae69713be2b9e0581a4388e9e144.diff) (3416 bytes) - added by adiroiban on 2014-02-27 13:55:12Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7001 7001
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__ 
branch__branches_pyflakes_tap_manhole_7001 branches/pyflakes-tap-manhole-7001
branch_author__radix radix
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1393509072000000 1393509072000000
changetime__1395967604000000 1395967604000000
version__None None
owner__radix radix

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban commented|
|-|-|

Just simple fixes of unused imports and variables.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix commented|
|-|-|

(In [42001]) Branching to pyflakes-tap-manhole-7001.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix set owner to @radix|
|-|-|

Thanks for the contribution!

The patch looks good, and passes all the builds. I'll merge it.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set status to `closed`|
|-|-|

(In [42003]) Merge pyflakes-tap-manhole-7001

Author: adiroiban
Reviewer: radix
Fixes: [#7001](https://github.com/twisted/twisted/issues/7001)

Clean up pyflakes warnings in twisted/manhole/ and twisted/tap/.
